### ISSUE TITLE
docker image version mismatch

### ISSUE THREAD

**Issue by apaul7** [Ensembl/ensembl-vep#427]
Hi!
I am trying to use the vep docker images hosted on docker hub but I think there is a mismatch with the tagged released version and the actual version in the image. 

the docker image tag `release_93.7` has vep version 94.2
the docker image tag `release_93.6` has vep version 94.1
the docker image tag `release_93.5` has vep version 93.5

From what I can tell this repo has the correct version and subversion for those tagged releases. But that does not match the image hosted on docker hub. Could you force a rebuild for the 93.6 and 93.7 images? I hope that's all that's needed!

Thank you,
-Alex

**Comment by ens-lgil** [comment]
Dear @apaul7,

Thank you for reporting this issue. 
This was due to an issue with the Dockerfile which cloned the `default` branch when it built these images and I believe that the release/94 was already set as the default branch when the Docker images `release/93.6` and `release/93.7` were built.
We fixed this issue on the DIockerfile and I am rebuilding the 2 images.
I will let you know when they will be available on DockerHub.

Best regards,
Laurent
